topics/statistics/tutorials/age-prediction_with_ml/tutorial.md

```diff
@@ -0,0 +1,368 @@
+>            - *"Select the primary metric (scoring):"*: `Regression -- 'r2'`
+>            - *"Select the cv splitter:"*: `KFold`
+>                - *"n_splits"*: `5`
+>                - *"Whether to shuffle data before splitting"*: `Yes`
+>                - *"Random seed number"*: `3111696`
+>            - *"Raise fit error:"*: `No`
+>    - *"Select input type:"*: `tabular data`
+>        - {% icon param-files %} *"Training samples dataset:"*: `train_rows` tabular file
+>        - *"Does the dataset contain header:"*: `Yes`
+>        - *"Choose how to select data by column:"*: `All columns BUT by column header name(s)`
+>            - *"Type header name(s):"*: `Age`
+>        - {% icon param-files %} *"Dataset containing class labels or target values"*: `train_rows` tabular file
+>        - *"Does the dataset contain header:"*: `Yes`
+>        - *"Choose how to select data by column:"*: `Select columns by column header name(s)`
+>            - *"Type header name(s):"*: `Age`
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. What is the optimal number of estimators for the given dataset?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. 75
+> >
+> {: .solution}
+
+## Predict age
+
+Using the hyperparameter search, we found the best model based on the training data. Now, we will predict age in the test dataset using this model in order to see if the model has learned important features which can generalise on a new dataset. The test dataset (`test_rows`) contains the same number of features but does not contain the `age` column. This is predicted using the trained model.
+
+> ### {% icon hands_on %} Hands-on: Predict age
+>
+> 1. **Ensemble methods for classification and regression** {% icon tool %} with the following parameters:
+>    - *"Select a Classification Task:"*: `Load a model and predict`
+>        - {% icon param-files %} *"Models"*: `Zipped` file (output of **Hyperparameter Search** {% icon tool %})
+>        - {% icon param-files %} *"Data (tabular)"*: `test_rows` tabular file
+>        - *"Does the dataset contain header:"*: `Yes`
+>        - *"Select the type of prediction:"*: `Predict class labels`
+>
+{: .hands_on}
+
+## Create plots
+
+In the previous step, we generated predictions for the test dataset. We have one more dataset (`test_rows_labels`) which contains the true age values of the test set. Using the true and predicted values of age in the test set, we will verify the performance by analysing the plots.
+
+> ### {% icon hands_on %} Hands-on: Create regression plots
+>
+> 1. **Plot actual vs predicted curves and residual plots of tabular data** {% icon tool %} with the following parameters:
+>    - {% icon param-files %} *"Select input data file :"*: `test_rows_labels` tabular file
+>    - {% icon param-files %} *"Select predicted data file :"*: `Tabular` file (output of **Ensemble methods for classification and regression** {% icon tool %})
+>
+>    > ### {% icon comment %} Comment
```

Comment:
You could move all the description of the outputs and the figures outside of the hands-on box (which should mostly serve to describe sth the reader has to do)